app/views/movies/index.html.slim

```diff
@@ -6,16 +6,33 @@ h1 Movies list
 == yield
 
 - if @movies.any?
-  table#items
-    - @movies.each do |movie|
-      tr
-        td.name = movie.name
-        td.year = movie.year
-        td = link_to 'Edit', edit_movie_path(movie)
+  div.row#items
+    - @movies.in_groups_of(4).each do |group|
+      .row.movierow
+        -group.each do |movie|
+          - if !movie.nil?
+
+              .col-md-3
+
+                -if !movie.image.nil? and !movie.image.thumb.nil? and !movie.image.thumb.url.nil?
```

Comment:
This long if statement could be moved out of the view.
 - model
 - view helper